src/composing/platt_scaling.rs

```diff
@@ -43,14 +44,14 @@ pub struct Platt<F, O> {
 }
 
 /// Parameters for Platt's Newton method
-pub struct PlattParams<F, O> {
+pub struct PlattParams<F, O, const C: bool> {
```

Comment:
It'd make sense to give this type param a more specific name such as `Check`/`CHECK` (or whatever the convention is for const generics).

Reply:
yes, I also thought about using an enum `IsChecked` to make it more descriptive

Reply:
You could also rename the const generic itself (not sure if that's idiomatic)